build.ts

```diff
@@ -31,12 +31,29 @@ const accents = [
 ] as const;
 
 async function makeThemeObject(
-  name: string,
-  accent: keyof Labels<Color, AlphaColor>,
-  palette: Labels<Color, AlphaColor>
+  identifier: string,
+  accent: AccentName,
+  palette: CatppuccinFlavor,
```

Comment:
We usually refer to `CatppuccinFlavor['colors']` as `palette`, this should just be called `flavor`.
```suggestion
  flavor: CatppuccinFlavor,
```